sympy/matrices/common.py

```diff
@@ -2019,6 +2019,59 @@ def replace(self, F, G, map=False, simultaneous=True, exact=None):
         return self.applyfunc(
             lambda x: x.replace(F, G, map=map, simultaneous=simultaneous, exact=exact))
 
+    def rot90(self, k=1):
+        """Rotates Matrix by 90 degrees
+
+        Parameters
+        ==========
+
+        k : int
+            Specifies how many times the matrix is rotated
+            anticlockwise.
```

Comment:
```suggestion
            by 90 degrees (clockwise when positive), counter-clockwise when negative).
```
then the rest of the bullet points can be eliminated